pkg/releaseutil/manifest.go

```diff
@@ -39,7 +39,7 @@ func SplitManifests(bigFile string) map[string]string {
 	// Basically, we're quickly splitting a stream of YAML documents into an
 	// array of YAML docs. In the current implementation, the file name is just
 	// a place holder, and doesn't have any further meaning.
-	tpl := "manifest-%d"
+	tpl := "manifest-%010d"
```

Comment:
This would be considered a breaking change as the expected output has been modified. The zero-index used to be `manifest-0`, but now it'll be `manifest-0000000000`. Users may have calling code expecting the former.

Additionally, that means this function has been limited to sort a maximum of 10 billion manifests, whereas before it could sort `math.MaxInt`. Not a big deal as I don't think anyone was sorting that amount of manifests, but regardless it's still a behavioural change.

A non-breaking change would be to replace `sort.Strings` with `sort.Sort` in `SortManifests`, and implement a sorter for the output of `SplitManifests`. That way, `sort.Sort` sorts the manifests in the expected order during SortManifests.

Reply:
Applied as suggested, sorting by the part after `manifest-`.

In addition, I changed the kind sorter to not sort by name (if other attributes are equal), since that would be against the idea of keeping the original file order as suggested in the PR.